state/reboot.go

```diff
@@ -84,6 +84,9 @@ func (m *Machine) clearFlag() error {
 
 	docID := m.doc.DocID
 	count, err := reboot.FindId(docID).Count()
+	if err != nil {
```

Comment:
I am not sure if this is needed here. I think the original code was written to ignore err since if there is an error, the count will be 0....

Reply:
Then the code should be `count, _ := reboot.FindId(docID).Count()`